main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -815,6 +815,8 @@ object Weeder2 {
         case TreeKind.Expr.Use => visitExprUseExpr(tree)
         case TreeKind.Expr.Literal => visitLiteralExpr(tree)
         case TreeKind.Expr.Apply => visitApplyExpr(tree)
+        case TreeKind.Expr.Index => visitIndexExpr(tree)
```

Comment:
Keep alphabetical

Reply:
I put them "alphabetical" but the rest of the list is sorted by occurrence I think?